src/KodKod.Analyzer/ApiActionsAreAttributeRoutedFixProvider.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+
+namespace KodKod.Analyzer
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(ApiActionsAreAttributeRoutedFixProvider)), Shared]
```

Comment:
Put the attributes on their own lines.

Also I don't believe you need the name if it's already decorating the type